src/sentry/integrations/slack/integration.py

```diff
@@ -16,12 +16,22 @@ class SlackIntegration(OAuth2Integration):
     oauth_authorize_url = 'https://slack.com/oauth/authorize'
     oauth_client_id = options.get('slack.client-id')
     oauth_client_secret = options.get('slack.client-secret')
-    oauth_scopes = (
+    oauth_scopes = tuple(sorted((
         'bot',
         'chat:write:bot',
         'commands',
+        'links:read',
+        'links:write',
         'team:read',
-    )
+    )))
+
+    def get_config(self):
```

Comment:
unused atm

Reply:
also this config will need to be pulled into the slack event bits, and this should also be true by default